ios/noiseScore/Info.plist

```diff
@@ -13,11 +13,11 @@
 	<key>CFBundleInfoDictionaryVersion</key>
 	<string>6.0</string>
 	<key>CFBundleName</key>
-	<string>$(PRODUCT_NAME)</string>
```

Comment:
NIT: I also ran into issues with this, wasn't sure what a final fix is, but we may want to at least document in the README that this should be updated before a release is made, same thing for `MARKETING_VERSION`.

Reply:
Sounds good. I'll get on that